geomstats/information_geometry/multivariate_normal.py

```diff
@@ -0,0 +1,348 @@
+"""Statistical Manifold of multivaraite normal distributions with the Fisher metric.
```

Comment:
Yes good point ! To be consistent with the paper, we should replace "Statistical manifold" with "Information manifold". We made this choice of terminology because in differential geometry, statistical manifold usually refers to a manifold with dually flat affine structure, which is not implemented in geomstats (yet).